server/hedley/modules/custom/hedley_migrate/hedley_migrate.module

```diff
@@ -392,29 +341,6 @@ function _hedley_migrate_process_column(array &$data, array &$locator, $index, a
   $data[$index][$row[$index]] = $locator[$locator_index] = $term->tid;
 }
 
-/**
```

Comment:
@AronNovak why is this removed?

Reply:
Same thinking here. If it's still in use, let me know where and how it's invoked.